src/javascript/app/pages/trade/barriers.js

```diff
@@ -160,14 +161,27 @@ const Barriers = (() => {
         Defaults.remove('barrier', 'barrier_high', 'barrier_low');
     };
 
+    /**
+    * Validate Barriers
+    */
     const validateBarrier = () => {
-        const barrier_element = getElementById('barrier');
-        const empty           = isNaN(parseFloat(barrier_element.value)) || parseFloat(barrier_element.value) === 0;
+        const barrier_element      = getElementById('barrier');
+        const empty                = isNaN(parseFloat(barrier_element.value))||parseFloat(barrier_element.value) === 0;
+        const barrier_high_element = getElementById('barrier_high');
+        const barrier_low_element  = getElementById('barrier_low');
+        const error_node           = getElementById('barrier_high_error');
```

Comment:
we can define this inside the `if (isVisible(barrier_high_element)) {` on line 181 or so